# HG changeset patch
# User sbr <sebastien@brissaud.name>
# Date 1375075763 -7200
#      Mon Jul 29 07:29:23 2013 +0200
# Node ID 7d5654037dacafa1b3b082500ceeeb8ee301f3ee
# Parent  3c74e9f9b3f8e66e13fdc3c202732963b1965010
-Add: CTRL+ALT+T as hotkey to toggle text direction

diff --git a/src/toolbar_gui.cpp b/src/toolbar_gui.cpp
--- a/src/toolbar_gui.cpp
+++ b/src/toolbar_gui.cpp
@@ -1663,6 +1663,7 @@ enum MainToolbarHotkeys {
 	MTHK_ZOOMEDIN_SCREENSHOT,
 	MTHK_DEFAULTZOOM_SCREENSHOT,
 	MTHK_GIANT_SCREENSHOT,
+	MTHK_TOGGLE_TEXT_DIRECTION,
 	MTHK_CHEATS,
 	MTHK_TERRAFORM,
 	MTHK_EXTRA_VIEWPORT,
@@ -1748,6 +1749,7 @@ struct MainToolbarWindow : Window {
 			case MTHK_ZOOMEDIN_SCREENSHOT: MenuClickLargeWorldScreenshot(SC_ZOOMEDIN); break;
 			case MTHK_DEFAULTZOOM_SCREENSHOT: MenuClickLargeWorldScreenshot(SC_DEFAULTZOOM); break;
 			case MTHK_GIANT_SCREENSHOT: MenuClickLargeWorldScreenshot(SC_WORLD); break;
+			case MTHK_TOGGLE_TEXT_DIRECTION: ToggleTextDirection(); break;
 			case MTHK_CHEATS: if (!_networking) ShowCheatWindow(); break;
 			case MTHK_TERRAFORM: ShowTerraformToolbar(); break;
 			case MTHK_EXTRA_VIEWPORT: ShowExtraViewPortWindowForTileUnderCursor(); break;
@@ -1853,6 +1855,7 @@ static Hotkey maintoolbar_hotkeys[] = {
 	Hotkey(WKC_CTRL  | 'P', "zoomedin_screenshot", MTHK_ZOOMEDIN_SCREENSHOT),
 	Hotkey(WKC_CTRL  | 'D', "defaultzoom_screenshot", MTHK_DEFAULTZOOM_SCREENSHOT),
 	Hotkey((uint16)0, "giant_screenshot", MTHK_GIANT_SCREENSHOT),
+	Hotkey(WKC_CTRL | WKC_ALT | 'T', "toggle_text_direction", MTHK_TOGGLE_TEXT_DIRECTION),
 	Hotkey(WKC_CTRL | WKC_ALT | 'C', "cheats", MTHK_CHEATS),
 	Hotkey('L', "terraform", MTHK_TERRAFORM),
 	Hotkey('V', "extra_viewport", MTHK_EXTRA_VIEWPORT),
